diff --git a/node_modules/react-native-vector-icons/react-native.config.js b/node_modules/react-native-vector-icons/react-native.config.js
index 356b254..635a773 100644
--- a/node_modules/react-native-vector-icons/react-native.config.js
+++ b/node_modules/react-native-vector-icons/react-native.config.js
@@ -1,5 +1 @@
-module.exports = {
-  dependency: {
-    assets: ['Fonts'],
-  },
-};
+module.exports = { assets: [] }
